### PR TITLE
[deps](log4j) upgrade log4j

### PR DESCRIPTION
**Pull request by hf200012**
upgrade log4j
Upgrade Log4j to version 2.15.0 to solve remote execution vulnerabilities
## Proposed changes

Close related #issue (replace it with issue number if it exists).

Describe the overview of changes, and introduce why we need it.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zbtzbtzbt** [comment]
The bug is belong to apache Log4j 2, not Log4j.

**Comment by tinkerrrr** [comment]
Module `fe_plugins` and `apache_hdfs_broker` should upgrade too.

**Comment by hf200012** [comment]
> The bug is belong to apache Log4j 2, not Log4j.

We are using this
org.apache.logging.log4j

> The bug is belong to apache Log4j 2, not Log4j.

We are using this
org.apache.logging.log4j
2 refers to version 2.x, we are using version 2.14.0, which is within the scope of influence

File: fe_plugins/pom.xml [hunk 1]
```diff
@@ -82,21 +82,21 @@ under the License.
...
-                <version>2.12.1</version>
+                <version>2.15.0</version>
             </dependency>
 
             <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-slf4j-impl -->
             <dependency>
                 <groupId>org.apache.logging.log4j</groupId>
                 <artifactId>log4j-slf4j-impl</artifactId>
-                <version>2.12.1</version>
+                <version>2.15.0/version>
```
Comment (morningman) [inline fe_plugins/pom.xml:99]:
```suggestion
                <version>2.15.0</version>
```

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**